xml/cl.xml

```diff
@@ -1383,6 +1385,17 @@ server's OpenCL/api-docs repository.
         <enum bitpos="2"            name="CL_DEVICE_KERNEL_CLOCK_SCOPE_SUB_GROUP_KHR"/>
             <unused start="3" end="63"/>
     </enums>
+    <enums name="cl_image_tiling_ext" vendor="Multiple">
+            <unused start="0" end="0"/>
+        <enum value="1"             name="CL_IMAGE_TILING_LINEAR_EXT"/>
+        <enum value="2"             name="CL_IMAGE_TILING_OPTIMAL_EXT"/>
+            <unused start="3" end="0xFFFFFFFF"/>
```

Comment:
Hmm, is this intentionally a separate value range, parallel to everything else?

Reply:
Yes. There are other examples for this pattern (see `cl_command_buffer_state_khr`, `cl_command_termination_reason_arm`, etc). Does it create a concrete problem?